sycl/test/vitis/simple_tests/single_task_vector_add.cpp

```diff
@@ -4,7 +4,7 @@
 // RUN: %clangxx -std=c++20 -fsycl -fsycl-targets=%sycl_triple %s -o %t.dir/exec.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.dir/exec.out
 // Test that we can get a fast reports by adding --vitis-ip-part
-// RUN: %clangxx -std=c++20 -fsycl -fsycl-targets=%sycl_triple %s -o %t.dir/exec2.out --vitis-ip-part=xc7vx330t-ffg1157-1
+// RUN: %clangxx -std=c++20 -fsycl -fsycl-targets=%sycl_triple %s -o %t.dir/exec2.out --vitis-ip-part=xcu200-fsgd2104-2-e
```

Comment:
What we should do here is figure out inside lit what part id is available on the system and setup a substitution for it. similarly to what we do for "%opencv4_flags"

Reply:
I think it is another problem.
Not everybody has an Alveo U200 platform installed but the `xcu200-fsgd2104-2-e` part comes de facto with Vivado.
But anyway, having the part should be configurable too.

Reply:
if xcu200-fsgd2104-2-e comes by default with Vitis/Vivado its fine keeping it.